delete_users.py

```diff
@@ -0,0 +1,71 @@
+#!/bin/python3
+# /// script
+# requires-python = ">=3.12"
+# dependencies = [
+#     "python-dotenv",
+#     "requests",
+# ]
+# ///
+"""
+Delete a set of users from the Mozilla Sentry organization.
+
+The user ids of the users to delete are read from stdin. Each line starting with a
+integer user id specifies a user to delete. Other lines are ignored. You can optionally
+specify the main email address of the user account in addition to the user id, and the
+script will only delete the user if the email address matches. Further fields after the
+email address are also ignored.
```

Comment:
Can you provide an example input in both cases?

E.g. list of user ids only:
```
1
2
3
```
E.g. list of user id and email address together
```
1 a@b.c
2 b@c.d
3 c@d.e
```